Local War/Assets/Resources/Script/Character/MovementSys.cs

```diff
@@ -66,10 +66,11 @@ private void move()
         /*****Move the player*****/
         controller.Move(moveDirection * Time.deltaTime);    //move the character based on the gravitational force.
         if (!controller.isGrounded)
-        {
             jumpDirection.y -= gravity * Time.deltaTime;
-        }
+
         controller.Move(jumpDirection * Time.deltaTime);
+        if(controller.isGrounded)
+            jumpDirection.y = 0;
```

Comment:
In the future, this really should be done by character systems.

Reply:
Do you mean the isGround function? Someone is already working on our own version of the function.

Reply:
yeah. if we merge this, they are probably going to have a merge conflict. it will only be one or two lines, but it's okay for now